src/core/opamConsole.ml

```diff
@@ -962,9 +962,13 @@ let print_table ?cut oc ~sep table =
   List.iter (fun l -> print_line (cleanup_trailing l)) table
 
 let menu ?default ?unsafe_yes ?yes ~no ~options fmt =
-  assert (List.length options < 10);
+  assert (List.length options < 36);
```

Comment:
It needs also a change in the function description
```diff
--- a/src/core/opamConsole.mli
+++ b/src/core/opamConsole.mli
@@ -132,7 +132,7 @@ val confirm:
     options are set.
     [no] is the option to choose otherwise, when non interactive, on [escape].
     [default] is the option to choose on an active empty input ("\n").
-    Max 9 options. *)
+    Max 35 options. *)
 val menu:
   ?default:'a -> ?unsafe_yes:'a -> ?yes:'a -> no:'a ->
   options:('a * string) list ->

Reply:
done